packages/app/src/components/Visualization/StartScreen.js

```diff
@@ -67,7 +67,7 @@ const StartScreen = ({ error, classes }) => {
                 {mostViewedVisualizations.length > 0 && (
                     <div style={styles.section}>
                         <h3 style={styles.title}>
-                            Most viewed charts and tables
+                            Your most viewed charts and tables
```

Comment:
This should also be translated with `i18n`!

Reply:
Indeed. Fixed